app/jenkins-for-jira-ui/src/components/JenkinsSetup/JenkinsSetup.tsx

```diff
@@ -354,30 +353,38 @@ const JenkinsSetup = (): JSX.Element => {
 				</div>
 			) : (
 				<>
-					<ConnectionFlowServerNameSubHeader serverName={serverName} />
 					<div className={cx(serverNameFormOuterContainer)}>
 						<div className={cx(connectionFlowInnerContainer)}>
 							<div className={cx(jenkinsSetupContainer)}>
 								<h3 className={cx(jenkinsSetupHeader)}>Set up Jenkins</h3>
-								<p className={cx(jenkinsSetupContent)}>
-									A Jenkins admin needs to set up your server to complete this connection.
-									Select the appropriate option:
-								</p>
+								<div className={cx(jenkinsSetupContent)}>
+									<p>
+										To complete the connection for <strong>{serverName}</strong>,
+										you'll need to set up your Jenkins
+										server and enter the webhook URL and secret we'll provide.
```

Comment:
Is this the copy? It seems like something is wrong here

Reply:
This is the context that the design people provide. I will bring this up in the project sync next week.